pkg/mcp/snapshot/snapshot.go

```diff
@@ -31,6 +33,16 @@ type Snapshot interface {
 	Version(collection string) string
 }
 
+// Info is used for configz
+type Info struct {
+	// Collection is type_url
+	Collection string
```

Comment:
This is not a type_url anymore. (If we have such references, we should remove them.